index.js

```diff
@@ -12,3 +14,36 @@ const pagination = document.querySelector('[data-js="pagination"]');
 const maxPage = 1;
 const page = 1;
 const searchQuery = "";
+
+async function fetchCharacters() {
+  try {
+    const response = await fetch("https://rickandmortyapi.com/api/character");
+    if (response.ok) {
+      const data = await response.json();
+      const results = data.results;
+      cardContainer.innerHTML = "";
+      results.forEach((person) => {
```

Comment:
Please stick to the naming convention 👉🏼 your parameter should be the singular of your array. As `result` is not the best choice, I would recommend to rename the array to `persons` or `characters`.
```suggestion
     characters.forEach((character) => {
```